content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Istio CA Integration with SPIRE 
+linktitle: How to integrate Istio with SPIRE through Envoy SDS API.
+description: Describes how to configure Istio to integrate with SPIRE to get cryptographic identities through Envoy's SDS API.
+weight: 40
+keywords: [kubernetes,spiffe,spire]
+aliases:
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+[SPIRE](https://spiffe.io/docs/latest/spire-about/spire-concepts/) is a production-ready implementation of the SPIFFE specification that performs node and workload attestation in order to securely
+issue cryptographic identities to workloads running in heterogeneous environments. SPIRE can be configured as a source of cryptographic identities for Istio workloads through an integration with
+[Envoy's SDS API](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret). Istio can detect the existence of a UNIX domain socket that implements the Envoy SDS API on a defined
+socket path, allowing Envoy to communicate and fetch identities directly from it.
+
+This integration with SPIRE provides flexible attestation options not available with the default Istio identity management while harnessing Istio's powerful service management.
+For example, SPIRE's plugin architecture enables diverse workload attestation options beyond the Kubernetes namespace and service account attestation offered by Istio.
+SPIRE's node attestation extends attestation to the physical or virtual hardware on which workloads run.
+
+For a quick demo of how this SPIRE integration with Istio works, see [Integrating SPIRE as a CA through Envoy's SDS API](https://github.com/istio/istio/tree/master/samples/security/spire).
+
+## Install SPIRE
+
+### Option 1: Quick start
+
+Istio provides a basic sample installation to quickly get SPIRE up and running:
+
+{{< text bash >}}
+$ kubectl apply -f @samples/security/spire/spire-quickstart.yaml
+{{< /text >}}
+
+This will deploy SPIRE into your cluster, along with two additional components: the [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi) — used to share the SPIRE Agent's UNIX domain socket with the other

```

Comment:
Could you do a search and replace in this doc for "UNIX domain socket"? Different people have suggested different capitalization forms. I guess we should go with "UNIX Domain Socket" as that is what the [Envoy SDS doc](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret) uses.

Don't forgot to fix the other line break issue I mentioned at the end of this sentence:

> SPIRE Agent can be configured to push federated bundles to Envoy through the Envoy SDS API, allowing Envoy to use [validation context](https://spiffe.io/docs/latest/microservices/envoy/#validation-context) to verify peer certificates and trust a workload from another trust domain.

After these two changes, I have no more suggestions.

Reply:
Okay, I will follow this pattern for the UDS.

There is another line break? Let me fix it, will check again as well.